docs/databases/connecting.md

```diff
@@ -154,6 +158,14 @@ To forget the data that Metabase has stored from previous [database scans](#sync
 
 ![Re-scan options](./images/re-scan-options.png)
 
+### Syncing and scanning using the API
```

Comment:
I moved this from the Troubleshooting guide. Do we want to keep API info in the reference doc here?

Reply:
I think it's okay here, as this topic is sufficiently advanced, and we don't go into too much detail. If more detail is required, we should include that in the endpoint docs themselves.